src/main/java/at/hannibal2/skyhanni/config/features/garden/TrackingConfig.kt

```diff
@@ -0,0 +1,170 @@
+            COOKIE_BUFF("§dCookie Buff", "Cookie <:booster_cookie:1263204080940220498>"),
+            GOD_POTION("§cGod Potion", "God Potion <:god_potion:1263204732390871151>"),
+            JACOBS_CONTEST("§eJacob's Contest", "Contest <:hoe:1263206591218585640>"),
+            ACTIVE_CROP("§aCrop", "Crop <:hoe:1263206591218585640>"),
+            ANITA_BUFF("§6Anita Buff", "Anita Buff <:anitas_artifact:1263212366330335376>"),
+            BPS("§eBlocks/Second", "BPS <:bps_sugar:1263285905083465729>"),
+            FARMING_SINCE("§fFarming Since", "Farming For <:minecraftclock:1264539139911716866>");
+
+            override fun toString(): String = display
+        }
+
+        @Expose
+        @ConfigOption(name = "Skin Part", desc = "Skin Part to be displayed (image) in the top right of the embed.")
+        @ConfigEditorDropdown
+        var bodyPart: SkinBodyPart = SkinBodyPart.HEAD
+
+        @Expose
+        @ConfigOption(
+            name = "Use Default",
+            desc = "Use default Discord color scheme for the embed sidebar" +
+                "(Online = Green, Idle = Yellow, Offline = Red).",
+        )
+        @ConfigEditorBoolean
+        var useDefault: Boolean = true
+
+        @Expose
+        @ConfigOption(name = "Embed Color", desc = "Which color the embed sidebar should be (Chroma displays as black).")
+        @ConfigEditorDropdown
+        var color: LorenzColor = LorenzColor.YELLOW
+    }
+
+    @Expose
+    @ConfigOption(name = "Message Type", desc = "Choose which way the status will be sent.")
+    @ConfigEditorDropdown
+    var messageType: MessageType = MessageType.NEW_MESSAGE
+
+    enum class MessageType(private val str: String) {
+        NEW_MESSAGE("New Message"),
+        EDITED_MESSAGE("Edited Message");
+
+        override fun toString(): String {
+            return str
+        }
+    }
+
+    @Expose
+    @ConfigOption(name = "Tracking", desc = "Send an embed with the options you selected above to your specified webhook.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    var tracking: Boolean = true
+
+    enum class Pet(val petName: String) {
+        BEE("Bee <:bee:1263201131064983673>"),
+        CHICKEN("Chicken <:chicken:1263201132658823178>"),
+        ELEPHANT("Elephant <:elephant:1263201134466830357>"),
+        MOOSHROOM_COW("Mooshroom Cow <:mooshroom_cow:1263201135884374026>"),
+        PIG("Pig <:pig:1263201137188802651>"),
+        RABBIT("Rabbit <:rabbit:1263201138371727421>"),
+        SLUG("Slug <:slug:1263201140086931511>"),
+        ;
+
+        override fun toString(): String {
+            var cleanName = name.replace("_", " ").lowercase(Locale.getDefault())
+            cleanName = cleanName.substring(0, 1).uppercase(Locale.getDefault()) + cleanName.substring(1)
+            if (cleanName == "Mooshroom cow") cleanName = "Mooshroom Cow"
+            return cleanName
+        }
+    }
+
+    enum class Crop(val display: String, val emoji: String) {
```

Comment:
can we use a map or function instead? there is no direct connection to the existing crop enum in garden api